lib/Completion/Bridge/TolerantParser/WorseReflection/KeywordCompletor.php

```diff
@@ -14,6 +14,26 @@
 
 class KeywordCompletor implements TolerantCompletor
 {
+    private const MAGIC_METHODS = [
+        '__call' => "(string \\\$\${1:name}, array \\\$\${2:arguments}): \${3:mixed}\n{\$0\n}",
+        '__callStatic' => "(string \\\$\${1:name}, array \\\$\${2:arguments}): \${3:mixed}\n{\$0\n}",
+        '__clone' => "(): void\n{\$0\n}",
+        '__construct' => "(\$1)\n{\$0\n}",
+        '__debugInfo' => "(): array\n{\$0\n}",
+        '__destruct' => "(): void\n{\$0\n}",
+        '__get' => "(string \\\$\${1:name}): \${3:mixed}\n{\$0\n}",
+        '__invoke' => "(\$1): \${2:mixed}\n{\$0\n}",
+        '__isset' => "(string \\\$\${1:name}): bool\n{\$0\n}",
+        '__serialize' => "(): array\n{\$0\n}",
+        '__set' => "(string \\\$\${1:name}, mixed \\\$\${2:value}): void\n{\$0\n}",
+        '__set_state' => "(array \\\$\${1:properties}): object\n{\$0\n}",
+        '__sleep' => "(): array\n{\$0\n}",
+        '__toString' => "(): string\n{\$0\n}",
+        '__unserialize' => "(array \\\$\${1:data}): void\n{\$0\n}",
+        '__unset' => "(string \\\$\${1:name}): void\n{\$0\n}",
+        '__wakeup' => "(): void\n{\$0\n}",
+    ];
```

Comment:
hmm, I basically did _not_ include these because `__construct` is probably the 99% use case, would it make sense to prioritize that?

![image](https://github.com/phpactor/phpactor/assets/530801/95f4ac05-014e-4237-8867-717715b25f9d)

for example brings it to the top of the list?

Reply:
I didn't look into the image, let me set special priority.